test/pytest/test_torch_compile.py

```diff
@@ -146,3 +176,90 @@ def test_serve_inference(self):
                 "Compiled model with backend inductor, mode reduce-overhead"
                 in model_log
             )
+
+    def test_compile_inference_enable_true_default(self, chdir_example):
```

Comment:
You should parametrize these tests to make it more obvious what is the difference between them. See https://github.com/pytorch/serve/blob/36049cb7f8046b5e72b0a790a1b0aa5b0e96aee6/test/pytest/test_example_gpt_fast.py#L98 for an example

Reply:
This is cool. Done